.claude/hooks/session-start.sh

```
@@ -0,0 +1,45 @@
+  # cng-toolkit is required at runtime but lives outside the lockfile; install
+  # it editable into the ingestion venv (matches CI setup).
+  uv pip install -e "../geo-conversions[all]"
+)
+
+echo "[session-start] Installing MCP deps (uv)..."
+( cd mcp && uv sync --extra dev --frozen )
+
+# Expose helpful env vars for the rest of the session.
+if [ -n "${CLAUDE_ENV_FILE:-}" ]; then
+  {
+    echo 'export PATH="$HOME/.local/bin:$PATH"'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)